### DISCUSSION TITLE
월드 입장 중 강제 종료 현상

### DISCUSSION THREAD

**Discussion by esang28** [naverz/zepeto-studio-kor#904] [5. Q&A]
안녕하세요, 제목과 같은 문제가 발생해 깃헙에 글을 올립니다.
제페토 월드를 제작하고 심사에 올렸다가 거절되었는데 내용이 다음과 같습니다.

'최저 사양의 아이폰(6S)과 안드로이드 폰(Galaxy S6)에서 월드 입장 도중 제페토 앱이 강제 종료 됩니다.'

가지고 있는 아이폰에서 작동하는데엔 무리가 없었어서, 최저 사양 구동 확인을 해보지 않았습니다만,
처음 월드에 입장할 때 불러오는 리소스가 많아서 강제 종료가 되는 것이라 예상하는데 이와 관련해 해결책이 있을지 여쭙니다.

**Comment by Sonak0930** [comment]
혹시 해결하셨나요? 저도 비슷한 현상을 겪고 있습니다.

**Reply by esang28** [reply]
네, 리소스 과다와 UI 쪽 스크립트 문제였던 것으로 보입니다. 첫 월드 로드 때 순간과부화로 어플이 비정상 종료되었던 것 같습니다.
스크립트 수정, 하이어라키에 사용하지 않는 오브젝트 정리, 그래픽 이미지 용량을 줄였더니 개선되었습니다.

**Reply by Sonak0930** [reply]
답변주셔서 감사합니다. 덕분에 해결했어요 ㅠㅠㅠ

**Reply by Sonak0930** [reply]
혹시 다른 것도 여쭤볼 수 있을까요..?

**Reply by esang28** [reply]
제가 디자이너라 이쪽으로 식견이 좁은 탓에 혹시 코드작성 관련이면 답변이 어렵습니다. 궁금하신 부분이 아는 내용이라면 답변 드리겠습니다~

**Comment by Sonak0930** [comment]
답장주셔서 감사합니다. 어차피 저도 제페토 1도 몰라서 괜찮습니다.
혹시 player transform  동기화 문제가 발생하지는 않으셨나요? 예를들어 플레이어 A B가 있는데 A를 다른 지점으로
순간이동 시켰더니 B 화면에선 A가 가만히 있고, A 화면에서만 이동하는 이런 버그들이요.
제가 알아본 바에 따르면 기본적인 움직임은 clientStarter에서 update가 되는 걸로 알고있는데 플레이어가 임의로 옮기는
경우에는 걷기 전까지 업데이트가 되지 않더라구요.
혹시 비슷한 경험이 있으실까요?

---------- Forwarded message ---------
보낸사람: esang28 ***@***.***>
Date: 2022년 7월 15일 (금) 오후 12:46
Subject: Re: [naverz/zepeto-studio-kor] 월드 입장 중 강제 종료 현상 (Discussion #904)
To: naverz/zepeto-studio-kor ***@***.***>
Cc: Jaemin ***@***.***>, Comment ***@***.***>


제가 디자이너라 이쪽으로 식견이 좁은 탓에 혹시 코드작성 관련이면 답변이 어렵습니다. 궁금하신 부분이 아는 내용이라면 답변 드리겠습니다~

—
Reply to this email directly, view it on GitHub
<https://github.com/naverz/zepeto-studio-kor/discussions/904#discussioncomment-3151930>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AOIF4D7XGYJWFASNZCJY2OTVUDNH7ANCNFSM5ZKK7ISQ>
.
You are receiving this because you commented.Message ID:
***@***.***>

**Reply by esang28** [reply]
해당 문제를 픽스해보지 않았어서 답을 드리기 어렵네요. 요 게시글들을 참고해보심이 어떨까요?
https://github.com/naverz/zepeto-studio-kor/discussions/672
https://github.com/naverz/zepeto-studio-kor/discussions/598

**Comment by Sonak0930** [comment]
아 그리고 이 주소가 메일 보내는 게 맞을까요? 깃헙 답변 링크인 거 같은데 제대로 보내고 있는지 모르겠네요

2022년 7월 15일 (금) 오후 3:01, Stone Rolling ***@***.***>님이 작성:

**Comment by Sonak0930** [comment]
안녕하세요 답변 감사합니다. 현재 서버 동기화 example을 찾아보면서 공부하고 있는데 쉽지 않은 거 같아요. 이전에 작업하실 때
월드 심사 제출해 보신 적 있나요? 공홈에서 월드 가이드라인이 있는데, capacity 제한이 있는지 모르겠네요

2022년 7월 19일 (화) 오후 2:31, esang28 ***@***.***>님이 작성:
